submissions/ayokofemi/navigation/Contact.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>CONTACT</title>
+</head>
+<body>
+    <h1>Contact Form</h1>
+    <form action="#" method="post" enctype="multipart/form-data">
+        <label for="Address">Address:</label>
+        <textarea name="Address" id="Address" cols="20" rows="5"></textarea><br>
+        <label for="Phone Number">Phone Number:</label>
+        <input type="text" name="Phone Number" id="Phone Number"><br>
+        <label for="email:">Email:</label>
+        <input type="email" name="email:" id="email:" placeholder="Enter your email address">
```

Comment:
I think you should not include symbol `:` to `name` and ` id` attributes, for now, to avoid confusion later on. Rectify the same on line 14